package.json

```diff
@@ -46,6 +46,7 @@
   },
   "dependencies": {
     "@wordpress/components": "^22.1.0",
+    "cypress-mochawesome-reporter": "^3.4.0",
```

Comment:
I think this should probably be in devDependencies for this repo alongside cypress. In this repo we include the react-character-map from the dependencies in the built version so there may be a risk we include this in the build unintentionally.

Are you able to double check?